powerdnsadmin/routes/admin.py

```diff
@@ -1873,7 +1925,7 @@ def delete_template(template):
             if result['status'] == 'ok':
                 history = History(
                     msg='Deleted domain template {0}'.format(template),
-                    detail=str({'name': template}),
+                    detail=str(json.dumps({'name': template})),
```

Comment:
@jbe-dw I think using `json.dumps` is the proper way, because a string always could contain a character that should be probably escaped. `json.dumps` takes care of that. Without proper escaping code like this https://github.com/PowerDNS-Admin/PowerDNS-Admin/pull/1160/commits/44c9aff5dba929876dc6a8c6007f90fd1070617f#diff-b7bf2cb7f2cef97121c5a2596b5d9f6578df2b7881fc4582e6ffede05024707eR552 throws an error. For example when an user puts in `' '` or other variations with quotation marks for login the history/dashboard which shows login attempts will crash with a 5** error.

Reply:
I meant that the function call is done inside a str() cast. The later one looks useless to me, as json.dumps outputs strings. This is what I asked, json.dumps() is clearly the proper way.